p2p/src/swarm/mod.rs

```diff
@@ -177,8 +175,13 @@ where
     /// The event is received from the networking backend and it's either a result of an incoming
     /// connection from a remote peer or a response to an outbound connection that was initiated
     /// by the node as result of swarm maintenance.
-    async fn accept_connection(&mut self, info: net::types::PeerInfo<T>) -> crate::Result<()> {
-        log::debug!("{}", info);
+    async fn accept_connection(
+        &mut self,
+        address: T::Address,
+        info: net::types::PeerInfo<T>,
+    ) -> crate::Result<()> {
+        let peer_id = info.peer_id;
+        log::debug!("peer {} connected, address {}, {}", peer_id, address, info);
```

Comment:
```suggestion
        log::debug!("peer {peer_id} connected, address {address}, {info}");
```
Nitpick: you don't like captured identifiers? 🙃

Reply:
I'll update it, thanks!